### ISSUE TITLE
How can I change the default loss 

### ISSUE THREAD

**Issue by MinuteswithMetrics** [amaiya/ktrain#259]
First I would like to say that ktrain is awesome and very helpful! 

I was wondering is it possible to change the default loss? For example, I would like to change it to SparseCategoricalCrossentropy.  

Would that be done here: `ktrain.get_learner()` or would that be in `model.compile`?

**Comment by amaiya** [comment]
Thanks - glad you found it helpful.  

See [this FAQ entry for how to use custom loss functions and optimizers](https://github.com/amaiya/ktrain/blob/master/FAQ.md#how-do-i-use-custom-loss-functions-or-optimizers).  You can just recompile the model with the loss function you want prior to calling `ktrain.get_learner`.   

Note that, for most tasks like text classification and image classification, integer (and string) labels are automatically reformatted to be 1-hot-encoded  when `ktrain` preprocesses your data (i.e., labels are currently formatted for use with categorical crossentropy).

**Comment by MinuteswithMetrics** [comment]
Thank you! I suspected that was the case for text classification, but I was unsure.
